setup.py

```diff
@@ -30,17 +30,21 @@ def read_requirements(path):
     with open(path, "r") as f:
         requirements = f.read().splitlines()
         processed_requirements = []
-
         for req in requirements:
             # For git or other VCS links
             if req.startswith("git+") or "@" in req:
-                pkg_name = re.search(r"(#egg=)([\w\-_]+)", req)
-                if pkg_name:
-                    processed_requirements.append(pkg_name.group(2))
-                else:
-                    # You may decide to raise an exception here,
-                    # if you want to ensure every VCS link has an #egg=<package_name> at the end
-                    continue
+                # check if "egg=" is present in the requirement string
```

Comment:
Was there a package in the requirements that needed this?

Reply:
bittensor revolution, it will go away once we merge it into master, however.